src/sentry/tasks/summaries/daily_summary.py

```diff
@@ -84,19 +93,19 @@ def schedule_organizations(timestamp: float | None = None, duration: int | None
             for user_option in users_with_tz:
                 users_by_tz[user_option.value].append(user_option.user_id)
 
-            # if it's ~4pm (btwn 4 and 5) for any of the users, generate the report
-            # we'll check the timezone for the users again before sending
-            time_to_send_timezones = []
-            for user_tz in users_by_tz.keys():
+            # if it's ~4pm (btwn 4 and 5) for any of the users, generate the report for them
```

Comment:
I got a Slack notification at 4:03pm and at 5:02pm today... maybe beat schedule timing / server times aren't so perfect that this could happen pretty regularly? Maybe it makes sense to track (in redis) if there was a delivery for the day to avoid duplicate notifications.

Reply:
Sorry about that! This was meant to be an internal only release for now but something went wrong 😬 I'll fix up the timing (and other bugs).